pages/1.10/release-notes/1.10.6/index.md

```diff
@@ -13,7 +13,7 @@ DC/OS 1.10.6 was released on April  24, 2018.
 [button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
 
 DC/OS 1.10.6 includes the following:
-- Apache Mesos 1.4.2-dev[change log](https://github.com/apache/mesos/blob/24d3886/CHANGELOG)
+- Apache Mesos 1.4.2-24d388[change log](https://github.com/apache/mesos/blob/24d3886/CHANGELOG)
```

Comment:
Hi @anithamuthu79  - I did a mistake. Let's go with 7 characters.  `1.4.2-24d3886` here and through out.